### PR TITLE
Add Pre reqs heading

### LINKED ISSUE #34: register.py doesn't work in Python 2
```shell
Enter your github username:0atman
Performing GitHub OAuth
This creates a personal access token with read:user and read:org scopes. 
Please enter your GitHub 2FA code:
Requesting authorization from GitHub - You will be prompted for your GitHub password.
Enter host password for user '0atman':
Traceback (most recent call last):
  File "./install.py", line 43, in <module>
    register("test")  # Remove test mode argument for production
  File "/code/register.py", line 66, in register
    except json.JSONDecodeError:
AttributeError: 'module' object has no attribute 'JSONDecodeError'
```

Looks like in Python 2, json decoding errors were `ValueError`s.

### PR DESCRIPTION
**Pull request by 0atman**
Fixes https://github.com/alphagov/gds-pre-commit/issues/34

### CODE REVIEW

**Review by danjoneslf — COMMENTED** [review]
Maybe we need some introductory text to explain that you can install the hooks without the registration. 

The TOTP 2FA should also be in here.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+**Hook**
+ - Python 2 or 3 with pip
+ 
+**Registration script**
+ - Python 3
+
+The pre-commit framework is installed by pip, the python package manager. To install it on your machine either brew install it or get it from your distributions package manager (it might be called `python3-pip`, or `pip3`)
+
+Currently, our registration script's github oauth flow only works with OTP keys (such as ubikeys), not TOTP keys. If you use such a key, please skip registration.
```
Comment (danjoneslf) [inline README.md]: This text is not correct. We do support TOTP like Google Authenticator / Authy. We don't support YubiKey's Fido U2F (https://www.yubico.com/authentication-standards/fido-u2f/)

**Review by akinnane — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## Pre-requisites
+
+**Hook**
```
Comment (seaemsi) [inline README.md]: We should only use bold for UI text. I recommend using a H3 here or just normal text.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+ - Python 2 or 3 with pip
+ 
+**Registration script**
+ - Python 3
+
+The pre-commit framework is installed by pip, the python package manager. To install it on your machine either brew install it or get it from your distributions package manager (it might be called `python3-pip`, or `pip3`)
```
Comment (seaemsi) [inline README.md]: We should capitalise Python

File: README.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+
+Currently, our registration script's github oauth flow only works with OTP systems (such as Google authenticator or authy or yubikeys that type in a code). If you use a U2F system, please skip registration.
```
Comment (seaemsi) [inline README.md]: GitHub OAuth flow

Comment (seaemsi) [inline README.md]: Google Authenticator, Authy, YubiKeys

File: README.md [hunk 5]
```diff
@@ -1,3 +1,15 @@
...
+
 ## Quick Install
 
 If you are happy with the default location, just run the following two commands to install:
```
Comment (seaemsi) [inline README.md]: We should avoid words like 'just' and 'simply'

File: pre-commit-plugins.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+## Useful hooks/plugins
+
+**Please note that these hooks are found on the pre-commit website but have not all been tested**
```
Comment (seaemsi) [inline pre-commit-plugins.md]: Try to avoid using bold for non-UI text

**Review by seaemsi — CHANGES_REQUESTED** [review]

**Pull request merged**